src/styles/stars.css

```
@@ -0,0 +1,38 @@
+.stars {
+  display: flex;
+  margin-bottom: 0;
+}
+
+.stars--1 :nth-child(-n + 1) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--3 :nth-child(-n + 3) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--5 :nth-child(-n + 5) {
+  background-image: url(../images/star-active.svg);
+}
```

you can join these selectors as it sets the same property with the same value
```suggestion
.stars--1 :nth-child(-n + 1),
.stars--2 :nth-child(-n + 2),
etc.... {
  background-image: url(../images/star-active.svg);
}
```